backend/Dockerfile

```diff
@@ -1,19 +1,16 @@
-FROM strapi/base:alpine
-
-ARG STRAPI_VERSION
-RUN yarn global add @strapi/strapi@${STRAPI_VERSION}
-
-WORKDIR /srv/app
-
-COPY ./package*.json ./
-RUN npm install
-
-COPY ./src/ ./src/
-COPY ./config/ ./config/
-COPY ./database/ ./database/
-COPY ./public/ ./public/
-COPY ./favicon.ico ./
-
-RUN npm run build
-
-CMD ["npm", "start"]
\ No newline at end of file
+FROM node:16
```

Comment:
Worth pointing out:
Switching from alpine to [buster](https://github.com/nodejs/docker-node/blob/6e8f32de3f620833e563e9f2b427d50055783801/16/buster/Dockerfile#L1) is going to build much larger (read: slower) docker images. We should try to prefer `alpine` when we can? We dont need to block this PR, but i hope we can switch back to alpine in the near future.

Reply:
Good point 👍 I'm still working on the PR, so I'll try to use also alpine